axon/neuromod.go

```diff
@@ -99,17 +99,17 @@ type NeuroModParams struct {
 func (nm *NeuroModParams) Defaults() {
 	// nm.DAMod is typically set by BuildConfig -- don't reset here
 	nm.DAModGain = 0.5
-	nm.DALrateMod.SetBool(false)
-	nm.AChLrateMod.SetBool(false)
-	nm.DALratePct = 0.5
-	nm.AChLratePct = 0.5
+	nm.DALRateMod.SetBool(false)
+	nm.AChLRateMod.SetBool(false)
+	nm.DALRatePct = 0.5
+	nm.AChLRatePct = 0.5
 	nm.BurstGain = 1
 	nm.DipGain = 1
 }
 
 func (nm *NeuroModParams) Update() {
-	mat32.Clamp(nm.DALratePct, 0, 1)
```

Comment:
This is the bug fix: this code was a no-op before, the return value was being ignored.